doc/classes/SpinBox.xml

```diff
@@ -13,23 +26,29 @@
 			<return type="LineEdit">
 			</return>
 			<description>
+				Returns the [LineEdit] id of the [SpinBox].
 			</description>
 		</method>
 	</methods>
 	<members>
 		<member name="align" type="int" setter="set_align" getter="get_align" enum="LineEdit.Align">
+			Sets the text alignment of the [SpinBox].
 		</member>
 		<member name="editable" type="bool" setter="set_editable" getter="is_editable">
+			If [code]true[/code], the [SpinBox] will be editable. Otherwise, it will not. Default value is [code]true[/code]
```

Comment:
I would say "Otherwise, it will be read-only."
You can skip the mention of the default value as #29380 will soon be merged and document all those automatically.